webapp/src/app.tsx

```diff
@@ -47,10 +47,11 @@ if (Utils.isDesktop() && Utils.isFocalboardPlugin()) {
         }
 
         const pathName = event.data.message?.pathName
-        if (!pathName) {
+        if (!pathName || !pathName.startsWith((window as any).frontendBaseURL)) {
             return
         }
 
+        Utils.log(`Navigating Boards to ${pathName}`)
```

Comment:
Is this new log entry still necessary?

Reply:
I wanted to keep this as it could help us figure out similar issues in the near future.